lib/screens/MyCard.dart

```diff
@@ -0,0 +1,119 @@
+import 'package:flutter/material.dart';
+import 'package:profile_page/widgets/card_row.dart';
+
+
+class my_card extends StatelessWidget {
+  const my_card({super.key});
+
+  @override
+  Widget build(BuildContext context){
+    return Stack(
+      children: <Widget>[
+        Column(
+          crossAxisAlignment: CrossAxisAlignment.stretch,
+        mainAxisAlignment: MainAxisAlignment.start,
+        children: <Widget>[
+          Container(
+            width: MediaQuery.of(context).size.width,
+            height: 140,
+            color: Color.fromARGB(255, 37, 70, 47),
+            
+        ),
+        Container(
+          width: MediaQuery.of(context).size.width,
+            height: 558,
+            color: Color.fromARGB(255, 207, 227, 213),
+           
+        )
```

Comment:
```suggestion
        Expanded(
                child: Container(
              width: MediaQuery.of(context).size.width,
              height: MediaQuery.of(context).size.height,
              color: Color.fromARGB(255, 207, 227, 213),
            )),
```
having specific height makes it less flexible to bigger screens so it's better to use MediaQuery.of(context).size.height and wrap the container with an Expanded widget to ensure it takes up all available space within its parent widget, which should prevent overflow issues